docker-compose.yml

```diff
@@ -6,7 +6,17 @@ services:
     image: maven:3.9.9-amazoncorretto-17-al2023
     environment:
       - ORG_SAGEBIONETWORKS_REPOSITORYSERVICE_ENDPOINT=${ORG_SAGEBIONETWORKS_REPOSITORYSERVICE_ENDPOINT}
-    command: mvn -Dorg.sagebionetworks.repositoryservice.endpoint=${ORG_SAGEBIONETWORKS_REPOSITORYSERVICE_ENDPOINT} gwt:run-codeserver
+      - DOCKER_BIND_IP
+    command: |
+      sh -c '
+        # If DOCKER_BIND_IP is set, pass it to the gwt-codeserver
+        BIND_ADDRESS_ARG=""
+        if [ -n "$DOCKER_BIND_IP" ]; then
+          BIND_ADDRESS_ARG="-Dgwt.bindAddress=$DOCKER_BIND_IP"
+        fi
+        # Run the GWT codeserver
+        mvn $$BIND_ADDRESS_ARG -Dorg.sagebionetworks.repositoryservice.endpoint=${ORG_SAGEBIONETWORKS_REPOSITORYSERVICE_ENDPOINT} gwt:run-codeserver
+      '
```

Comment:
If `DOCKER_BIND_IP` is provided, provide it as the `gwt.bindAddress` environment variable to support hosting the dev server on the network.

By default, do not host it (only accessible via localhost/127.0.0.1)